<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>797</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea_28">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_31">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>856</width>
         <height>732</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_212">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_NM1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_NM2">
            <item>
             <widget class="QLabel" name="label_NM2">
              <property name="text">
               <string>Fluid Lumen NodeSet</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="fluidset_NM1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>51</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_NM3">
            <item>
             <widget class="QLabel" name="label_NM2_2">
              <property name="text">
               <string>Fluid Database File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="fluidfile_NM2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>50</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_NM3">
              <property name="text">
               <string>choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_NM4">
            <item>
             <widget class="QLabel" name="label_NM4_4">
              <property name="text">
               <string>Fluid Porthole File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_NM3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_NM5">
              <property name="text">
               <string>choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_SaveFluidPloFile_NM">
            <item>
             <widget class="QLabel" name="label_SaveFluidPloFile_NM">
              <property name="text">
               <string>SaveFluidPloFile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_SaveFluidPloFile_NM">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_SaveFluidPloFile_NM">
              <property name="text">
               <string>choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NM5_2">
            <property name="text">
             <string>exort Fluid data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_NM5">
          <item>
           <widget class="QLabel" name="label_NM4">
            <property name="text">
             <string>Solid Lumen NodeSet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lumenset_NM">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>52</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_62">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_NM6">
            <item>
             <widget class="QLabel" name="label_NM6">
              <property name="text">
               <string>Solid Database File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEditNM5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_NM2">
              <property name="text">
               <string>choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_SaveSolidPloFile_NM">
            <item>
             <widget class="QLabel" name="label_SaveSolidPloFile_NM">
              <property name="text">
               <string>SaveSolidPloFile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_SaveSolidPloFile_NM">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_SaveSolidPloFile_NM">
              <property name="text">
               <string>choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NM7">
            <property name="text">
             <string>export Solid Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_NM1">
          <item>
           <widget class="QLabel" name="label_NM1">
            <property name="text">
             <string>ADINApath</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_NM1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>25</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NM1">
            <property name="text">
             <string>chooose</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_NM7">
          <item>
           <widget class="QPushButton" name="pushButton_NM3_2">
            <property name="text">
             <string>submit fd to Adina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NM">
            <property name="text">
             <string>Submit SD to Adina</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_NM2_2">
            <property name="text">
             <string>Strip Data</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_366">
          <item>
           <widget class="QLabel" name="label_PressureID_NM">
            <property name="text">
             <string>PressureID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_PressureID_NM">
            <property name="text">
             <string>90</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_EleFaceSetPrefix_NM">
            <property name="text">
             <string>EleFaceSetPrefix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_EleFaceSetPrefix_NM">
            <property name="text">
             <string>99</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_TimeFunctionPrefix_NM">
            <property name="text">
             <string>TimeFunctionPrefix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_TimeFunctionPrefix_NM">
            <property name="text">
             <string>9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_LoadPrefix_NM">
            <property name="text">
             <string>LoadPrefix</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_LoadPrefix_NM">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_74">
          <item>
           <widget class="QLabel" name="label_308">
            <property name="text">
             <string>inlet/outlet Set</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNM">
            <property name="text">
             <string>103,152,153</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_309">
            <property name="text">
             <string>offesetvalue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_NM2">
            <property name="text">
             <string>2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_310">
            <property name="text">
             <string>GroupNum</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_NM3">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Run_NM">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_207">
          <item>
           <widget class="QLabel" name="label_801">
            <property name="text">
             <string>Batch Render</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_497" stretch="1,10,1">
            <item>
             <widget class="QLabel" name="label_802">
              <property name="text">
               <string>TablePath</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPlainTextEdit" name="plainTextEdit_BatchTable_NM">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>25</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_BatchTable_NM">
              <property name="text">
               <string>Choose</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_BatchRun_NM">
            <property name="text">
             <string>BatchRender</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_31">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>876</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
